.github/workflows/ci-docs.yaml

```diff
@@ -33,6 +33,9 @@ jobs:
     steps:
       - run: echo "Skipping CI for docs"
   e2e-tests:
```

Comment:
Is there a specific reason of keeping "e2e-tests" in ci-docs.yaml?

Reply:
If we have a check that says "e2e-tests" need to pass for the repo, then there has to be a corresponding item in ci-docs that says "e2e-tests".

Reply:
In this case, not even e2e-tests enough, we have to ensure that all these test pass.
```
e2e-tests (self-hosted, macos, amd64, 11.7)
e2e-tests (self-hosted, macos, amd64, 12.6)
e2e-tests (self-hosted, macos, arm64, 11.7)
e2e-tests (self-hosted, macos, arm64, 12.6)
```